image-layout.md

```diff
@@ -163,16 +163,9 @@ Those tags will often be represented in an image-layout repository with matching
 ```json,title=Image%20Index&mediatype=application/vnd.oci.image.index.v1%2Bjson
 {
   "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
     {
-      "mediaType": "application/vnd.oci.image.index.v1+json",
-      "size": 7143,
-      "digest": "sha256:0228f90e926ba6b96e4f39cf294b2586d38fbb5a1e385c05cd1ee40ea54fe7fd",
-      "annotations": {
-        "org.opencontainers.image.ref.name": "stable-release"
-      }
-    },
```

Comment:
> I think we need fix mediatype description of manifest in image-index.md. It does not say `application/vnd.oci.image.index.v1+json` is supported.

That's because [compliant consumers don't *need* to be able to handle `application/vnd.oci.image.index.v1+json` at all][1].  And we do [require consumers to ignore unknown media types][2]

[1]: https://github.com/opencontainers/image-spec/blame/a82c09852aba4978e3d16bb0cd583d5e2f4a63c9/image-index.md#L4
[2]: https://github.com/opencontainers/image-spec/blame/a82c09852aba4978e3d16bb0cd583d5e2f4a63c9/image-index.md#L39